test/controllers/UKBankAccountControllerSpec.scala

```diff
@@ -64,6 +65,12 @@ class UKBankAccountControllerSpec extends ControllerWithQuestionPageBehaviours {
       identifiers.UKBankAccountId,
       true
     )
+
+    "set the IsAboutBankDetailsCompleteId to false to change to in progress status" in {
```

Comment:
This would read better if inside another clause, i.e. *when* must the complete id be set to false?